### DISCUSSION TITLE
BCL not working for me

### DISCUSSION THREAD

**Discussion by mh88888** [BuildingComponentLibrary/bcl-manifest#23] [General]
HI! I am trying to download measures from BCL through OpenStudio but when I access Online BCL, no measures are showing up. Also, when I go to this link from the BCL website: https://bcl.nrel.gov/browse , I am getting "internal server error". Is this a problem on my end (and how might I fix it) or is BCL down? Thanks!
![BCL](https://user-images.githubusercontent.com/118770041/203146668-b1402b74-b5f6-43eb-b6fc-491f611b14ef.PNG)

**Comment by kflemin** [comment]
Could you try again? I think I have resolved the problem and https://bcl.nrel.gov/browse is back online now.   thanks for letting us know!

**Reply by mh88888** [reply]
Yes, it is working! Thank you :)
